src/api/app/models/issue_tracker.rb

```diff
@@ -32,6 +33,18 @@ def write_to_backend
     self.issues_updated ||= Time.now
   end
 
+  def cve?
+    kind == "cve"
+  end
+
+  def verify_issue_name!(name)
```

Comment:
On Freitag, 10. Juni 2016, 04:37:05 CEST wrote Mois�s D�niz Alem�n:
> > @@ -32,6 +33,18 @@ def write_to_backend
> >      self.issues_updated ||= Time.now
> >    end
> >  
> > +  def cve?
> > +    kind == "cve"
> > +  end
> > +
> > +  def verify_issue_name!(name)
> 
> Nice! I love it, just one more thing.... don't kill me please :/
> 
> What about having the raise of the exception at patchinfo model instead at the method that validates? I mean, a method that validates only need to return a boolean. And then the patchinfo, that already raises another exception , will raise the proper exception.

that way?

-- 

Adrian Schroeter
email: adrian@suse.de

SUSE Linux GmbH, GF: Felix Imend�rffer, Jane Smithard, Graham Norton, HRB 21284 (AG N�rnberg) 
 
Maxfeldstra�e 5                         
90409 N�rnberg 
Germany